@xolotl/currentscape_multi.m

```diff
@@ -0,0 +1,181 @@
+% 
+% ### currentscape
+%
+% **Syntax**
+%
+% ```matlab
+% x.currentscape_multi()
+% [I_in, I_out, V] = x.currentscape_multi()
+% ```
+%
+% **Description**
+%
+% This method creates a [currentscape](https://elifesciences.org/articles/42722), or a visualization of the fractional
+% contribution of each current at each point in time 
+% 
+% When called with one output, no plot is created, but the raw normalized
+% inward and outward currents are returned. 
+%
+
+
+function varargout = currentscape_multi(self,plot_here)
+
+self.output_type=1;
+data = self.integrate;
+
+fn = fieldnames(data);
+
+for i = 1:numel(fn) % go over the compartments
+    fni = string(fn(i));
+    field{i} = data.(fni);
+    fn1 = fieldnames(field{i});
+    
+    k=1;
+    for j=4:numel(fn1) % go over the currents
+        fn1i = string(fn1(j));
+        
+        if isfield(field{i}.(fn1i),'I')==1
+        I{i}(:,k) = field{i}.(fn1i).I; % ionic currents
+        else
+        I{i}(:,k) = field{i}.(fn1i).state(:,1); % synaptic current   
+        end  
+        k=k+1;
+    end
+end
+
+for i=1:numel(I)
+    
+I_out{i} = I{i}; I_in{i} = I{i};
```

Comment:
why are currents being stored as cell arrays? a matrix is much faster and easier to work with